GoogleUtilities/Network/GULMutableDictionary.m

```diff
@@ -76,14 +76,18 @@ - (NSUInteger)count {
   return count;
 }
 
-- (id)objectForKeyedSubscript:(id<NSCopying>)key {
-  // The method this calls is already synchronized.
-  return [self objectForKey:key];
+- (void)setObject:(id)obj forKeyedSubscript:(id<NSCopying>)key {
+  dispatch_async(_queue, ^{
```

Comment:
I would prefer to keep old implementation here that reuses the method implementation rather than copying it. Despite that the implementation is minimal now, modifying it in the future will be easier.

Reply:
After discussion agreed to keep separate implementations because of the different requirements for nil keys.